src/modules/add.js

```diff
@@ -0,0 +1,15 @@
+export const tasks = JSON.parse(localStorage.getItem('array')) || [];
+
+export const add = () => {
+  const input = document.querySelector('.addinput');
+  const value1 = input.value.trim();
+  if ((value1.length !== 0)) {
+    const obj = {
```

Comment:
I appreciate how you're storing the value of input.value.trim() in a variable, but it would be better if you gave the variable a more descriptive name.